hepcrawl/spiders/elsevier.py

```diff
@@ -0,0 +1,188 @@
+import glob
+import os
+import shutil
+import tempfile
+import xml.etree.ElementTree as et
+import zipfile
+
+import boto3
+import requests
```

Comment:
this is wrong, you shouldn't have to use `requests`. The way `scrapy` works is that you yield a `Request`, which then gets consumed by the `scrapy` middleware, which downloads the response and passes it to the callback, see https://github.com/inspirehep/hepcrawl/blob/1d6c9c7b8716d5b98b5f604828a809b8f04b6e9e/hepcrawl/spiders/aps_spider.py#L104-L107 for an example. This allows `scrapy` to do all the scheduling of requests to allow doing requests in parallel, avoid overloading the server, etc.